homeassistant/components/notify/pushover.py

```diff
@@ -51,10 +51,13 @@ def send_message(self, message="", **kwargs):
         """Send a message to a user."""
         from pushover import RequestError
 
-        # Make a copy and use empty dict as default value (thanks @balloob)
-        data = dict(kwargs.get(ATTR_DATA, {}))
+        data_ref = kwargs.get(ATTR_DATA)
+
+        # Make a copy and use empty dict if necessary (thanks @balloob)
+        data = dict(data_ref) if data_ref else {}
```

Comment:
This code is exactly the same. Passing in a second argument to `get` specified the default value if the key is not found.